apps/wallet/src/ui/app/components/receipt-card/TxnAmount.tsx

```diff
@@ -11,9 +11,11 @@ type TxnAmountProps = {
     label: string;
 };
 
+// dont show amount if it is 0
```

Comment:
I know it's not a common use-case, but is there a reason we want to hide the amount when someone sends sui to themselves? amount still seems useful in activity feed to me


**this PR**
<img width="362" alt="image" src="https://user-images.githubusercontent.com/122397493/216703842-37b90594-8735-4443-bea1-99becd7d9750.png">

**main**
<img width="364" alt="image" src="https://user-images.githubusercontent.com/122397493/216703537-d302ab16-956c-45cb-a388-a562c1f11d56.png">

Reply:
> I know it's not a common use-case, but is there a reason we want to hide the amount when someone sends sui to themselves? amount still seems useful in activity feed to me
> 
> **this PR** <img alt="image" width="362" src="https://user-images.githubusercontent.com/122397493/216703842-37b90594-8735-4443-bea1-99becd7d9750.png">
> 
> **main** <img alt="image" width="364" src="https://user-images.githubusercontent.com/122397493/216703537-d302ab16-956c-45cb-a388-a562c1f11d56.png">

It is zero